### ISSUE TITLE
This Is The Only Level - Caps Lock not working

### ISSUE THREAD

**Issue by SicklySilverMoon** [ruffle-rs/ruffle#7002]
### Describe the bug

Stage 24 of this is the only level (`www.jmtb02.com/s/thisistheonlylevel.swf`) does not have the green gate/bar at the end of the level open when caps lock is held.

### Expected behavior

On stage 24 holding down caps lock should cause the green gate to be open as long as caps lock is held.

### Affected platform

Desktop app

### Operating system

5.17.6-1-MANJARO #1 SMP PREEMPT Mon May 9 07:52:43 UTC 2022 x86_64 GNU/Linux Release 21.2.6

**Comment by Herschel** [comment]
Tried briefly on my Windows machine and seemed to work, so possibly a Linux only issue.

**Comment by Herschel** [comment]
Indeed, seems like a winit issue on Linux. cc [winit#1864](https://github.com/rust-windowing/winit/pull/1864) (ty @Aaron1011)
